src/orquestra/sdk/_base/_driver/_models.py

```diff
@@ -438,7 +440,10 @@ class Message(pydantic.BaseModel):
     Single line content.
     """
 
-    ray_filename: RayFilename
+    # Pydantic v2 lets us do multiple aliases
```

Comment:
**thought**: so far, `Message` was only about WorkflowRunLogs entries (IIRC). Now, we're starting to use it for TaskRunLogs entries, which look similar as of today, but it might get messy when two endpoint schemas diverge.